src/gateway_config_led.erl

```diff
@@ -125,7 +125,7 @@ handle_info({lights_event, Event}, State=#state{}) ->
 
 handle_info(diagnostics_timeout, State=#state{}) ->
     Diagnostics = gateway_config:diagnostics(State#state.miner_proxy),
-    State1 = handle_led_event(p2p_led_event(Diagnostics), State),
+    State1 = handle_led_event(p2p_led_event(Diagnostics), State#state{diagnostics=Diagnostics}),
```

Comment:
Why is this now added to the state?

Reply:
it was a bug to leave it out, which was one of the two cause of the led color issue we were having